buildconfig/CMake/LinuxSetup.cmake

```diff
@@ -4,21 +4,21 @@
 include(DetermineLinuxDistro)
 
 # ##############################################################################
-# If required, find tcmalloc
+# If required, find jemalloc
 # ##############################################################################
-option(USE_TCMALLOC "If true, use LD_PRELOAD=libtcmalloc.so in startup scripts"
+option(USE_JEMALLOC "If true, use LD_PRELOAD=libjemalloc.so in startup scripts"
        ON
 )
 # If not wanted, just carry on without it
-if(USE_TCMALLOC)
-  find_package(Tcmalloc)
-  if(TCMALLOC_FOUND)
-    set(TCMALLOC_LIBRARY ${TCMALLOC_LIBRARIES})
-  endif(TCMALLOC_FOUND)
+if(USE_JEMALLOC)
+  find_package(Jemalloc)
```

Comment:
Thanks for changing the finder name but I think that now means this needs updating as I'm getting a warning about not being able to located the finder.
I think we should also make this mandatory at this point as the dev has requested it with `USE_JEMALLOC=ON` and if we don't find it I think we should say that loudly:

`find_package(JemallocLib REQUIRED)`

Reply:
Sorry, I probably forgot to properly clean cmake before testing. This time, it should be good.

Reply:
No problem. Thanks for making those changes. I'll test it out on CentOS now.